docker/utils/utils.py

```diff
@@ -952,6 +953,8 @@ def format_env(key, value):
         return '{key}={value}'.format(key=key, value=value)
     return [format_env(*var) for var in six.iteritems(environment)]
 
+def duration_from_string(duration_string):
+    return timeparse(duration_string) * 1000000000
```

Comment:
Yeah. We should also provide a utility function for building the object, following the pattern we have with the networking data structures (e.g. `create_ipam_pool`):

```python
def create_healthcheck(test=None, interval=None, timeout=None, retries=None):
    return {
        'Test': test,
        'Interval': interval,
        'Timeout': timeout,
        'Retries': retries,
    }
```